package.json

```diff
@@ -11,7 +11,7 @@
     "@types/react": "^16.9.53",
     "@types/react-dom": "^16.9.8",
     "enzyme": "^3.11.0",
-    "jest": "^27.0.4",
+    "jest": "26.6.0",
```

Comment:
I'm not sure If u really tested it, but this error is more nested into pakages. I tried fixing it and It requires a lot deps version updates.

First thing If I remember was then checking "@types/jest" and everything related to jest overall